### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by zoef-yt**
# Added products list page to E-com
Live preview:- [Products Page](https://deploy-preview-4--need-games-zoef.netlify.app/)

## Updates

- [x] Added products page
- [x] Added filter bar to products listing page
- [x] Added fake data using faker js
- [x] Created filter bar functionality to scroll using useState.
- [x] Added "add to wishlist" new icon SVG.

## What not to check
- [ ]  The functionality of the app
- [ ]  Routing
- [ ] Working of products button.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { Homepage } from './components/homepage/homepage';
-
+// import { Homepage } from './components/homepage/homepage';
```
Comment (zoef-yt) [inline src/App.jsx:2]: This is commented for routing to the products page of this pr.

File: src/components/css/products-page.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+}
+
+.dropdown-content > * {
+	margin: 1rem;
+	display: block;
+}
+
+.dropdown-content > input {
+	display: inline-block !important;
+}
```
Comment (srishtigautam21) [inline src/components/css/products-page.css]: Do not use !important

**Review by srishtigautam21 — APPROVED** [review]
Good to merge

File: src/components/css/products-page.css (hunk 2 above)
Comment (zoef-yt) [reply src/components/css/products-page.css]: Done removed.

**Review by j24m — APPROVED** [review]
Good work. :)

File: src/components/products-page/product-page-components/filter-bar.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { genres, types } from '../../../data/products-data';
+
+const FilterDropDownbutton = (props) => {
```
Comment (sudipkundu999) [inline src/components/products-page/product-page-components/filter-bar.jsx:4]: Destructuring of props can be done. (not necessary)

File: src/components/products-page/product-page-components/product-page-body.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+					<SingleProductCard
+						key={product.id}
+						image={product.image}
+						image2={product.image2}
+						name={product.name}
+						price={product.price}
+						offer={product.offer}
+						inStock={product.inStock}
+					/>
+				);
```
Comment (sudipkundu999) [inline src/components/products-page/product-page-components/product-page-body.jsx:19]: Rule of thumb: When more than 3 properties are there try destructuring. It'll look cleaner.
(Not important)

**Review by sudipkundu999 — APPROVED** [review]
UI and code looks great.

File: src/components/products-page/product-page-components/product-page-body.jsx (hunk 4 above)
Comment (zoef-yt) [reply src/components/products-page/product-page-components/product-page-body.jsx:19]: Will make this in the next update. Thanks!

**Pull request merged**